### PR TITLE
chore(deps): bump linux versions

### PR DESCRIPTION
**Pull request by Benedikt-Frenzel**
## Summary of Pull Request

Updated the minor versions for: 

- Debian 12
- Ubuntu 22.04
- Red Hat Enterprise Linux 9
- AlmaLinux 9
- Rocky Linux 9
- Oracle Linux 9
- Fedora Server

## Type of Pull Request

- [ ] This is a bugfix. `type/bug`
- [ ] This is an enhancement or feature. `type/feature` or `type/enhancement`
- [ ] This is a documentation update. `type/docs`
- [ ] This is a refactoring update. `type/refactor`
- [x] This is a chore. `type/chore`
- [ ] This is something else.
      Please describe:

## Related to Existing Issues

Issue Number: N/A

## Test and Documentation Coverage

- [x] Tests have been completed.
- [ ] Documentation has been added or updated.

## Breaking Changes?

- [ ] Yes, there are breaking changes.
- [x] No, there are no breaking changes.

### CODE REVIEW

File: project.json [hunk 1]
```diff
@@ -51,9 +51,9 @@
                                 "architectures": [
                                     {
                                         "architecture": "amd64",
-                                        "download_link": "https://mirror.cogentco.com/debian-cd/12.7.0/amd64/iso-cd/debian-12.7.0-amd64-netinst.iso",
+                                        "download_link": "https://mirror.cogentco.com/debian-cd/12.8.0/amd64/iso-cd/debian-12.7.0-amd64-netinst.iso",
```
Comment (tenthirtyam) [inline project.json]: Doesn't seem correct.

**Review by tenthirtyam — CHANGES_REQUESTED** [review]

**Comment by tenthirtyam** [comment]
Commits require the sign off.

**Comment by Benedikt-Frenzel** [comment]
I double checked and bumped all the URLs for:

- AlmaLinux
- Debian
- Fedora Server
- Rocky
- Ubuntu

**Comment by tenthirtyam** [comment]
Your commit still requires a sign-off.

File: .gitignore [hunk 2]
```diff
@@ -52,6 +52,10 @@ artifacts/**/*.nvram
 artifacts/**/*.ovf
 artifacts/**/*.vmdk
 
+## Ignore ISO path
```
Comment (tenthirtyam) [inline .gitignore]:
```suggestion
## Ignore ISO Path
```

File: .gitignore [hunk 3]
```diff
@@ -52,6 +52,10 @@ artifacts/**/*.nvram
...
+iso/
+
```
Comment (tenthirtyam) [inline .gitignore:57]:
```suggestion
```

File: project.json [hunk 4]
```diff
@@ -259,9 +259,9 @@
                                 "architectures": [
                                     {
                                         "architecture": "amd64",
-                                        "download_link": "https://dl.rockylinux.org/vault/rocky/9.4/isos/x86_64/Rocky-9.4-x86_64-dvd.iso",
+                                        "download_link": "https://dl.rockylinux.org/vault/rocky/9.5/isos/x86_64/Rocky-9.5-x86_64-dvd.iso",
                                         "checksum_algorithm": "sha256",
-                                        "checksum": "https://dl.rockylinux.org/vault/rocky/9.4/isos/x86_64/CHECKSUM"
+                                        "checksum": "https://dl.rockylinux.org/vault/rocky/9.5/isos/x86_64/CHECKSUM"
```
Comment (tenthirtyam) [inline project.json:264]: Not mentioned in the summary.

File: project.json [hunk 5]
```diff
@@ -320,9 +320,9 @@
                                 "architectures": [
                                     {
                                         "architecture": "amd64",
-                                        "download_link": "https://repo.almalinux.org/almalinux/9.4/isos/x86_64/AlmaLinux-9.4-x86_64-dvd.iso",
+                                        "download_link": "https://repo.almalinux.org/almalinux/9.5/isos/x86_64/AlmaLinux-9.5-x86_64-dvd.iso",
                                         "checksum_algorithm": "sha256",
-                                        "checksum": "https://repo.almalinux.org/almalinux/9.4/isos/x86_64/CHECKSUM"
+                                        "checksum": "https://repo.almalinux.org/almalinux/9.5/isos/x86_64/CHECKSUM"
```
Comment (tenthirtyam) [inline project.json:325]: Not mentioned in the summary.

File: project.json [hunk 6]
```diff
@@ -473,14 +473,14 @@
                     "versions": {
                         "40": [
                             {
-                                "version": "40",
+                                "version": "41",
```
Comment (tenthirtyam) [inline project.json:476]: This would need to be updated in the docs and `README.md`.

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
Each commit requires a sign-off.

For example, squash or amend each and included your version of the below based on your GitHub display name and the associated email address in your GitHub account.

```
Signed-off-by: Ryan Johnson <ryan.johnson@broadcom.com>
```

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
If v9 for both Rocky Linux and AlmaLinux are updated, then the same should be done for RHEL.

**Comment by Benedikt-Frenzel** [comment]
Updated the Readme and signed the commit. 
I cannot check and fix RHEL as I don't have a subscription, but I updated OL aswell.

File: README.md [hunk 7]
```diff
@@ -41,7 +41,7 @@ This project supports the following guest operating systems:
 | Oracle Linux                 | 8         |
 | CentOS Stream                | 9         |
 | CentOS Stream                | 8         |
-| Fedora Server                | 39        |
+| Fedora Server                | 41        |
```
Comment (tenthirtyam) [inline README.md:43]: Also here: https://github.com/vmware-samples/packer-examples-for-vsphere/blob/fb37d5e8488c5f253461c8328e7cad32202d76a5/docs/index.md?plain=1#L37

File: project.json (hunk 6 above)
Comment (tenthirtyam) [reply project.json:476]: https://github.com/vmware-samples/packer-examples-for-vsphere/blob/fb37d5e8488c5f253461c8328e7cad32202d76a5/docs/index.md?plain=1#L37

**Comment by tenthirtyam** [comment]
@Benedikt-Frenzel Please pull the latest. I've squashed and updated the commits.

File: project.json (hunk 6 above)
Comment (tenthirtyam) [reply project.json:476]: Resolved in https://github.com/vmware-samples/packer-examples-for-vsphere/commit/3cb90f4e13531fc53075a486a3852e6b6cdd4444.

**Review by tenthirtyam — CHANGES_REQUESTED** [review]
If v9 for Rocky Linux, AlmaLinux, and Oracle Linux are updated, then the same should be done for RHEL 9 but updating the checksum to the v9.5 release.

**Comment by Benedikt-Frenzel** [comment]
I just saw that I also should update the paths in the .hcl.example files, will do that later.

**Comment by tenthirtyam** [comment]
> `b3005f2`

Resol

> If v9 for Rocky Linux, AlmaLinux, and Oracle Linux are updated, then the same should be done for RHEL 9 but updating the checksum to the v9.5 release.

Resolved in b3005f2.

**Comment by tenthirtyam** [comment]
> I just saw that I also should update the paths in the .hcl.example files, will do that later.

Addressed in https://github.com/vmware-samples/packer-examples-for-vsphere/commit/691784217ec93e6d144869929514eb4dd69b93bd. Also needed an update to the path for fedora and the gitlab ci yaml.

**Comment by tenthirtyam** [comment]
Please pull the latest. I've squashed and updated the commits.

**Review by tenthirtyam — APPROVED** [review]
LGTM

**Pull request merged**